pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable_test.go

```diff
@@ -75,9 +74,9 @@ func TestNodeUnschedulable(t *testing.T) {
 	for _, test := range testCases {
 		nodeInfo := framework.NewNodeInfo()
 		nodeInfo.SetNode(test.node)
-
-		p, _ := New(nil, nil)
-		gotStatus := p.(framework.FilterPlugin).Filter(context.Background(), nil, test.pod, nodeInfo)
+		_, ctx := ktesting.NewTestContext(t)
+		p, _ := New(ctx, nil, nil)
```

Comment:
check the error here too

Reply:
Thanks for the catch! I checked the files again. Updated.